collect_app/src/main/res/layout/main_menu.xml

```diff
@@ -44,6 +43,8 @@
                 <org.odk.collect.android.views.MultiClickSafeButton
                     android:id="@+id/enter_data"
                     style="@style/Widget.Collect.Button.FullWidth"
+                    android:backgroundTint="?colorSurface"
```

Comment:
These buttons should really be `?colorPrimary` but they also shouldn't really be that giant. Down the line we should redesign the main menu interface, but for the moment I think sticking with the big grey buttons is the best idea.